lib/shopify_api/context.rb

```diff
@@ -167,6 +179,18 @@ def host_scheme
       def host_name
         T.must(URI(T.must(host)).host)
       end
+
+      private
+
+      sig { params(log_level: Symbol).returns(T::Boolean) }
+      def valid_log_level?(log_level)
+        return true if ::ShopifyAPI::Logger::LOG_LEVELS.keys.include?(log_level)
+
+        ShopifyAPI::Logger.warn("#{log_level} is not a valid log_level. "\
+          "Valid options are #{::ShopifyAPI::Logger::LOG_LEVELS.keys.join(", ")}")
```

Comment:
```suggestion
          "Valid options are #{::ShopifyAPI::Logger.levels.join(", ")}")
```
This an the above change help to decouple from the internals of the logger.